backend/apps/profile_info/tests/excluded/test_generate_data.py

```diff
@@ -0,0 +1,9 @@
+from apps.profile_info.factories import BaseUserFactory
+
+from unittest import TestCase 
+
+
+class GenerateDataTests(TestCase):
```

Comment:
Get rid of this, the communities one is just temporary to generate some data for the database, and this should be marked with @skip either way so the database isn't filled with 100 new users every time the tests are run.